### PR TITLE
MAAP onboarding guide

### PR DESCRIPTION
**Pull request by bsatoriu**
This PR is for the new onboarding doc: https://github.com/MAAP-Project/maap-documentation/blob/getting-started/docs/source/getting_started.ipynb 

@gchang @abarciauskas-bgse  I wanted to include the following text regarding the approval process, but wasn't sure if we had a support email. Is there an email address we can use here?

> At this point, a MAAP administrator will approve your account which will grant you access to the MAAP ADE. This process typically takes 5-10 minutes. To check on the status of your pending account, contact the MAAP team at **support@maap-project.org**.

### CODE REVIEW

**Comment by gchang** [comment]
Looks good, however, if someone followed the doc verbatim today, they would end up on the pilot ops site (but registering on Ops URS, which wouldn't work). I think we can release the doc today with it pointing to "*.ops.maap-project.org" ade and repo link, noting that this is temporary for now and update the docs later when we migrate hostnames.

**Comment by gchang** [comment]
The changes are good! @spa0002 any issues, stylistic or otherwise, with integrating it into readthedocs?

Also with the email, @abarciauskas-bgse, can we set up a support@maap-project.org email and forward it to multiple people?

File: docs/source/getting_started.ipynb [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+   "source": [
+    "### Uploading your public SSH key \n",
+    "\n",
+    "In order to access your ADE workspaces using SSH, you'll need to upload your public SSH key to your MAAP profile using the MAAP portal at https://ops.maap-project.org. \n",
+    "\n",
+    "Click on your profile name (or the \"Login\" button) in the top right corner shown here:\n",
+    "\n",
+    "![pyCMC mapview](_static/portal.png)\n",
+    "\n",
+    "On your profile page, click the \"Choose File\" button to upload your key,\n",
```
Comment (spa0002) [inline docs/source/getting_started.ipynb:68]: Recommend changing comma to period at the end of the sentence.

File: docs/source/getting_started.ipynb [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+   "source": [
+    "### Note on future URL changes\n",
+    "\n",
+    "The \"*.ops.maap-project.org\" URLs referenced in this guide will be converted to \"*.maap-project.org\" once the version one of MAAP is released in the near future. At that time, these URLs will be updated accordingly."
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
```
Comment (spa0002) [inline docs/source/getting_started.ipynb:85]: Appears there is an empty code block at the end of the page.

**Review by spa0002 — CHANGES_REQUESTED** [review]
Looks good. Recommend editing the index.rst file in the source directory so that the MAAP onboarding guide will appear in the table of contents. The table of contents is expected to change but for now can add the file name under the first `toctree` in index.rst file. For ex:

`Welcome to the MAAP User Documentation!
=======================================

.. toctree::
  :maxdepth: 2
  
  welcome.ipynb
  getting_started.ipynb
  git_clone_repo.ipynb`

Also, two minor edits recommended.

**Comment by bsatoriu** [comment]
@spa0002 thanks for the comments! Just committed those updates.

**Review by spa0002 — APPROVED** [review]

**Pull request merged**